source/renderer/app/components/widgets/TopLevelPopOver.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { PopOver, PopOverProps } from 'react-polymorph/lib/components/PopOver';
+
+const topLevelZIndex = 9999;
+
+const TopLevelPopOver = ({ children, ...props }: PopOverProps) => (
```

Comment:
Well done! We could even introduce the concept of layers instead of relying on numbers. e.g:

```
<TopLevelPopOver layer='normal' />
<TopLevelPopOver layer='foreground' />
```